backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/request/api/impl/RequestBL.java

```diff
@@ -166,6 +168,25 @@ public I_R_Request createRequestFromDDOrderLine(@NonNull final I_DD_OrderLine dd
 		return createRequest(requestCandidate);
 	}
 
+	@Override
+	public I_R_Request createRequestFromOrder(@NonNull final I_C_Order order)
+	{
+		final int requestTypeId = docTypeDAO.getById(order.getC_DocTypeTarget_ID()).getR_RequestType_ID();
```

Comment:
use order.getC_DocType_ID()

Reply:
and because we have 2 places with same issue... i suggest introducing a new method in OrderBL:

Optional<RequestTypeId> getRequestTypeForCreatingNewRequestsAfterComplete(I_C_Order order)